parm/config/config.resources

```diff
@@ -414,88 +414,107 @@ elif [ ${step} = "gldas" ]; then
     export npe_node_gaussian=$(echo "${npe_node_max} / ${nth_gaussian}" | bc)
     export is_exclusive=True
 
-elif [[ ${step} = "fcst" || ${step} = "efcs" ]]; then
+elif [[ "${step}" = "fcst" || "${step}" = "efcs" ]]; then
 
     export is_exclusive=True
 
-    if [[ ${step} == "fcst" ]]; then
+    if [[ "${step}" = "fcst" ]]; then
         _CDUMP_LIST=${CDUMP:-"gdas gfs"}
-    elif [[ ${step} == "efcs" ]]; then
+    elif [[ "${step}" = "efcs" ]]; then
         _CDUMP_LIST=${CDUMP:-"enkfgdas enkfgfs"}
     fi
 
     # During workflow creation, we need resources for all CDUMPs and CDUMP is undefined
     for _CDUMP in ${_CDUMP_LIST}; do
-        if [[ ${_CDUMP} =~ "gfs" ]]; then
+        if [[ "${_CDUMP}" =~ "gfs" ]]; then
           export layout_x=${layout_x_gfs}
           export layout_y=${layout_y_gfs}
           export WRITE_GROUP=${WRITE_GROUP_GFS}
           export WRTTASK_PER_GROUP=${WRTTASK_PER_GROUP_GFS}
           ntasks_fv3=${ntasks_fv3_gfs}
           ntasks_quilt=${ntasks_quilt_gfs}
-          nth_fv3=${nth_fv3_gfs}
+          nthreads_fv3=${nthreads_fv3_gfs}
         fi
 
         # PETS for the atmosphere dycore
-        (( FV3PETS = ntasks_fv3 ))
+        (( FV3PETS = ntasks_fv3 * nthreads_fv3 ))
+        echo "FV3 using (nthreads, PETS) = (${nthreads_fv3}, ${FV3PETS})"
```

Comment:
Do we want to keep all of these echos in?

Reply:
Yes.  It is good to see what the component wide domain decomposition is IMO. These don't get echoed out because they are encapsulated inside the `(( ... ))`